src/cairo_types/new_syscalls.rs

```diff
@@ -96,3 +97,14 @@ pub struct StorageWriteRequest {
 pub struct ReplaceClassRequest {
     pub class_hash: Felt252,
 }
+
+#[derive(Debug, Eq, PartialEq)]
+pub struct EcPointCoordinates {
+    pub x: BigUint,
+    pub y: BigUint,
+}
```

Comment:
This is not the right place to define this type. The `cairo_types` crate should only contain types that derive `FieldOffsetGetters`.